### DISCUSSION TITLE
Add testing against atlas (local deployment)

### DISCUSSION THREAD

**Discussion by hasezoey** [Automattic/mongoose#14261] [Ideas]
This is a issue to not forget about thinking to add a atlas CI test (local deployment)

https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-deploy-local/ (as of writing, it is in preview)

original: https://github.com/Automattic/mongoose/pull/14251#issuecomment-1892267271

**Comment by alexbevi** [comment]
If there are any questions/issues with the Atlas CLI when the team gets around to testing it for Atlas CI let me know.
